collections/xcube-gen-s2-mci.yaml

```diff
@@ -1,4 +1,5 @@
 Name: Maximum Chlorophyll Index (MCI)
+BYOAID: mean-ndvi
```

Comment:
Sorry for not spotting this earlier but I don't think this is a good/correct name for this algorithm. Where does it come from? For me it feels very counter-intuitive to the whole algorithm description.

Reply:
good catch, this is just my bad ID matching.

Reply:
in fact this algorithm is currently not available as it was associated with an account that is no longer available. needs clarification by @gunbra32